packages/shared/components/popups/MintNftConfirmationPopup.svelte

```diff
@@ -27,7 +40,25 @@
         ...(attributes && { attributes }),
     }
 
-    $: isTransferring = $selectedAccount.isTransferring
+    let nftTabDetails: { [key in string]: string }
+    $: {
+        nftTabDetails = {
+            name,
+            ...(description && { description }),
+            uri,
+            ...(issuerName && { issuerName }),
+            ...(collectionName && { collectionName }),
+        }
+    }
+
+    async function prepareNftOutput(): Promise<void> {
+        const outputData = buildNftOutputData(actualMintDetails, $selectedAccount.depositAddress)
```

Comment:
`actualMintDetails` doesn't provide a lot of information imo. What do you think about `irc27Metadata`?